### PR TITLE
add a clause to digging to make it obvious that you are use the digger

### PR DESCRIPTION
**Pull request by agoodman00**
Hey, one of my first changes in the base code rather than purely borg stuff :)

Not sure if you want this change or if I am messing something up (like if other code is parsing the dig message and I missed it.

### CODE REVIEW

File: src/cmd-cave.c [hunk 1]
```diff
@@ -527,6 +527,7 @@ static bool do_cmd_tunnel_aux(struct loc grid)
 	struct player_state local_state;
 	struct player_state *used_state = &player->state;
 	int oldn = 1, dig_idx;
+	char *with_clause = current_weapon == NULL ? "with your hands" : "with your weapon";
```
Comment (backwardsEric) [inline src/cmd-cave.c]: Add const before "char *" here.

**Comment by backwardsEric** [comment]
I don't have a strong opinion on whether the extra verbosity of the messages with this change outweighs the benefit of making it clear what's doing the digging.

**Comment by agoodman00** [comment]
It is also possible to add "temporarily switching to xxx to dig" instead of making all the messages more verbose.  I like the idea of it being obvious that digging with a digger in inventory uses that digger but I am not attached to any specific way of making that clear.

**Comment by NickMcConnell** [comment]
I say it's not too much more verbose, let's go with it.

**Pull request merged**